shared/actions/users.tsx

```diff
@@ -89,6 +89,7 @@ const submitRevokeVouch = async (_: TypedState, action: UsersGen.SubmitRevokeVou
     })
   } catch (e) {
     logger.info('error revoking vouch', e)
+    return
```

Comment:
How is an error supposed to be displayed in the UI here?
Is this another ticket?
cc @xgess

Reply:
great point. i have no idea what this is supposed to do. and there's not another ticket. nor is it in the designs. there isn't really a reason this should ever fail, so i'm inclined to let it black bar. fix coming.